clients/clickhouse-admin-keeper-client/src/lib.rs

```diff
@@ -2,11 +2,11 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Interface for making API requests to a clickhouse-admin server running in
+//! Interface for making API requests to a clickhouse-admin-{keeper|server} server running in
```

Comment:
Just `clickhouse-admin-keeper` here right?

Reply:
Fixed in [9862199](https://github.com/oxidecomputer/omicron/pull/6837/commits/98621999699a16737ee73df59b5f17c880dfcb7c)